tsconfig.base.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The `tsconfig.base.json` and `tsconfig.json` are under the `config` directory, so we are separating it?

Reply:
Commit by mistake. Removed.